### ISSUE TITLE
[Solved] [False Alarm] What happened to the Fade... feature? It's disappeared.

### ISSUE THREAD

**Issue by oscdat** [SubtitleEdit/subtitleedit#5756]
Right Click > ASAA Tools > Fade...
I checked the update notes but it didn't mention anything about removing Fade? Is it only me or did I miss an important announcement?

Thank you.

**Comment by MbuguaDavid** [comment]
In the Beta releases, the Fade option is not available by default. 
You'll need to download it as a Plugin from the file menu. 
Hope this helps. 
Thanks! 

https://user-images.githubusercontent.com/92556454/152282319-23f0e6d1-28bf-4404-983c-030d49ab0f20.mp4

**Comment by oscdat** [comment]
> In the Beta releases, the Fade option is not available by default. You'll need to download it as a Plugin from the file menu. Hope this helps. Thanks!
> 
>  Download.Fade.as.a.Plugin.mp4

Hi David, I hope it's really you. I just came here to change my post to "Solved" coz I just found that out for myself but then I saw your reply. And let me take this oportunity to tell you that I'm a big fan of yours, already subscribed. I've watched your entire playlist ( all 23 ) and each second was time well spent. 
And one more thing, before I made opened this thread. I checked your fade video ( coz that's how I knew such a thing even existed ) and in the comment section, you replied saying that, fade is available in 3.6.4 version. So I confirmed that something was off and immediately came here to post.
Thank you for all the tutorials and  and keep the videos coming. Much love.
